### ISSUE TITLE
CSS Grid

### ISSUE THREAD

**Issue by Snugug** [sasstools/sass-lint#1045]
There are a number of [CSS Grid properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) that are [not in `properties.yml`](https://github.com/sasstools/sass-lint/blob/develop/data/properties.yml). We should 2x check that property list with the full, final, shipped list from MDN and update accordingly

---

What version of Sass Lint are you using?
* 1.10.2, from CLI

Please include any relevant parts of your configuration
* ` - no-misspelled-properties: true`

What did you do? Please include the actual source code causing the issue.

```scss
grid-gap: 1em;
grid-row-gap: 1em;
```

What did you expect to happen?

* I expected no misspelled properties as they are [valid CSS Grid properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout)

What actually happened? Please include any error messages given to you by Sass Lint.

```
error  Property `grid-gap` appears to be spelled incorrectly      no-misspelled-properties
error  Property `grid-row-gap` appears to be spelled incorrectly  no-misspelled-properties
```
